net.mkiol.SpeechNote.yaml

```
@@ -0,0 +1,428 @@
+app-id: net.mkiol.SpeechNote
+default-branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+rename-desktop-file: dsnote.desktop
+rename-icon: dsnote
+rename-appdata-file: dsnote.metainfo.xml
+command: dsnote
+finish-args:
+    - --share=ipc
+    - --share=network
+    - --socket=x11
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --socket=pulseaudio
+    - --device=dri
+    - --filesystem=xdg-desktop
+    - --filesystem=xdg-download
+    - --filesystem=xdg-music
+    - --filesystem=xdg-documents
+    - --env=QT_QUICK_CONTROLS_STYLE=org.kde.desktop
+cleanup:
+    - '/cmake'
+    - '/etc'
+    - '/man'
+    - '/include'
+    - '/lib/debug'
+    - '/lib/fst'
+    - '/lib/cmake'
+    - '/lib/pkgconfig'
+    - '/lib/python3.10/site-packages/cmake*'
+    - '/share/cmake'
+    - '/share/doc'
+    - '/share/locale'
+    - '/share/man'
+    - '/share/pkgconfig'
+    - '/share/rhvoice'
+    - '/share/runtime'
+    - '/share/vim'
+    - 'libboost*'
+    - 'libRHVoice_audio*'
+    - 'pcre2*'
+    - 'xz*'
+    - 'unxz*'
+    - '*.a'
+    - '*.la'
+modules:
+    - name: fmt
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DFMT_DOC=OFF
+        - -DFMT_TEST=OFF
+        - -DFMT_FUZZ=OFF
+      sources:
+        - type: archive
+          url: https://github.com/fmtlib/fmt/releases/download/9.1.0/fmt-9.1.0.zip
+          md5: 6133244fe8ef6f75c5601e8069b37b04
+
+    - name: openblas
+      only-arches:
+        - x86_64
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DBUILD_TESTING=OFF
+        - -DBUILD_WITHOUT_LAPACK=OFF
+        - -DC_LAPACK=ON
+        - -DDYNAMIC_ARCH=ON
+        - -DBUILD_STATIC_LIBS=OFF
+        - -DBUILD_SHARED_LIBS=ON
+      sources:
+        - type: archive
+          url: https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21.tar.gz
+          md5: ffb6120e2309a2280471716301824805
+
+    - name: openblas
+      only-arches:
+        - aarch64
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DBUILD_TESTING=OFF
+        - -DBUILD_WITHOUT_LAPACK=OFF
+        - -DC_LAPACK=ON
+        - -DDYNAMIC_ARCH=ON
+        - -DBUILD_STATIC_LIBS=OFF
+        - -DBUILD_SHARED_LIBS=ON
+        - -DTARGET=ARMV8
+      sources:
+        - type: archive
+          url: https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21.tar.gz
+          md5: ffb6120e2309a2280471716301824805
```

this can be made ONE.

```suggestion
    - name: openblas
      buildsystem: cmake
      config-opts:
        - -DCMAKE_BUILD_TYPE=Release
        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
        - -DBUILD_TESTING=OFF
        - -DBUILD_WITHOUT_LAPACK=OFF
        - -DC_LAPACK=ON
        - -DDYNAMIC_ARCH=ON
        - -DBUILD_STATIC_LIBS=OFF
        - -DBUILD_SHARED_LIBS=ON
      build-options:
          arch:
              aarch64:
                  config-opts:
                      - -DTARGET=ARMV8
      sources:
        - type: archive
          url: https://github.com/xianyi/OpenBLAS/releases/download/v0.3.21/OpenBLAS-0.3.21.tar.gz
          md5: ffb6120e2309a2280471716301824805
```

---

also please don't use `md5` but `sha256`.

---

Fixed in 8df0e78